setup.cfg

```diff
@@ -48,11 +48,15 @@ markers=
 filterwarnings=
     error
     ignore::pytest.PytestUnraisableExceptionWarning
+    ignore:Save attribute '_xsection' was not loaded.:RuntimeWarning
```

Comment:
we shouldn't ignore this globally, better to fix the saved files to have the correct attribute or ignore it locally in the particular tests that cause it

Reply:
Also should add a default to the `set_up` method like we had talked about in a previous PR? Or include a default at the class level where the zernike cross section is defined?